src/appkit.cs

```diff
@@ -8794,7 +9023,7 @@ partial interface NSHelpManager {
 	[BaseType (typeof (NSObject), Delegates=new string [] { "WeakDelegate" }, Events=new Type [] { typeof (NSImageDelegate)})]
 	[Dispose ("__mt_reps_var = null;")]
 	[ThreadSafe]
-	partial interface NSImage : NSCoding, NSCopying, NSSecureCoding, NSPasteboardReading, NSPasteboardWriting {
+	partial interface NSImage : NSCopying, NSSecureCoding, NSPasteboardReading, NSPasteboardWriting {
```

Comment:
removal of `NSCoding` is a breaking change (unless some other protocol subclass it)

Reply:
It already had NSSecureCoding further in the list.